lib/tasks/annual_report.rake

```diff
@@ -0,0 +1,12 @@
+require "mail"
+# usage: 
+# rake MAILTO=your.email@gsa.gov FY=15 annual_report:annual_report
+
+namespace :annual_report do
+	desc "Emails FY approvals"
```

Comment:
Not sure if we have rubocop configured for this, but I believe we are indenting 2 spaces in this project (so this should be bumped out a bit)